.env

```
@@ -80,6 +82,12 @@ _APP_DOCKER_HUB_USERNAME=
 _APP_DOCKER_HUB_PASSWORD=
 _APP_CONSOLE_GITHUB_SECRET=
 _APP_CONSOLE_GITHUB_APP_ID=
+_APP_VCS_GITHUB_APP_NAME=
+_APP_VCS_GITHUB_PRIVATE_KEY=""
```

```suggestion
_APP_VCS_GITHUB_PRIVATE_KEY=
```
Any reason for not leaving it empty like the other variables?

---

Value is multiline. Double quotes make multiline variables work. It is an indication for you to use double quotes instead of just pasting the key.

Up to you, we can remove those quotes.

---

I'd say lets keep it consistent with the other vars. Using double quotes doesn't imply that the variable here has to be enclosed in double quotes.
We should instead make it explicitly known here
https://appwrite.io/docs/environment-variables